client/src/pages/AccountButton.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Button, Typography } from '@material-ui/core';
+
+const useStyles = makeStyles(theme => ({
+  button: {
+    margin: theme.spacing(1),
+  	"margin-right" : "2%",
+  },
+  input: {
+    display: 'none',
+  },
+  title: {
+  	"font-family" : theme.fontFamily,
+  	"font-size" : "10px",
+  	padding : "5px"
```

Comment:
indents and camel case for consistency!

Reply:
Thanks Ronald! This file has actually been deleted in the latest commit (I think it shows up when you view all commits), but I'll make the formatting changes for the rest of the file :)